index.html

```diff
@@ -0,0 +1,176 @@
+            </ul>
+            <p class="body-detail">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+              <ul class="courses">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+            <form action="#">
+                <button class="btn-body" type="submit" name="submit">see project</button>
+            </form>
+        </section>
+
+        <section class="work-one work">
+          <img class ="img-work" src="./templates/Snap3(6).svg" alt="Tonic">
+          <h2>Tonic</h2>
+            <ul class="work-history">
+              <li class="cover">CANOPY</li>
+              <li><img class="counter" src="./templates/Counter(1).svg" alt="counter"></li>
+              <li class="career">back end dev</li>
+              <li><img class="counter" src="./templates/Counter(1).svg" alt="counter"></li>
+              <li class="career">2015</li>
+            </ul>
+            <p class="body-detail">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+              <ul class="courses">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+            <form action="#">
+                <button class="btn-body" type="submit" name="submit">see project</button>
+            </form>
+        </section>
+
+        <section class="work-one work">
+          <img class ="img-work" src="./templates/Snap4(7).svg" alt="Tonic">
+          <h2>Multi-Post<br> Stories</h2>
+            <ul class="work-history">
+              <li class="cover">CANOPY</li>
+              <li><img class="counter" src="./templates/Counter(1).svg" alt="counter"></li>
+              <li class="career">back end dev</li>
+              <li><img class="counter" src="./templates/Counter(1).svg" alt="counter"></li>
+              <li class="career">2015</li>
+            </ul>
+            <p class="body-detail">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+              <ul class="courses">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+            <form action="#">
+                <button class="btn-body" type="submit" name="submit">see project</button>
+            </form>
+        </section>
+
+        <section class="about-myself">
+          <h1 class="myself-header">about myself</h1>
+          <p class="myself-paragraph">Hello I’m a software developer! I can help<br> you build a product , feature or website<br> Look through some of my work and<br> experience! If you like what you see and have a project you need coded, don’t<br> hestiate to contact me. </p>
+          <a class="link-up" href="#">LET'S CONNECT</a>
+          <nav class="icon-body">
```

Comment:
- [ ] I think it's better to remove the nav tag and only keep the ul. Because The <nav> HTML element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents within the webapp.